c_project/RadfordNeal.ppm

```diff
@@ -0,0 +1,116 @@
+P6
```

Comment:
@khorassa what is this file? If it isn't required, please remove it from the repo and merge

Reply:
Could be the image we send., although probably not a good idea for the demo. I'll remove it